reporters/default_reporter.go

```diff
@@ -176,11 +174,7 @@ func (r *DefaultReporter) DidRun(report types.SpecReport) {
 		}
 	case types.SpecStateFailed:
 		highlightColor, header = "{{red}}", fmt.Sprintf("%s [FAILED]", denoter)
```

Comment:
I think this line is leftover form an earlier merge?  You should be able to remove it as highlightColor is set elsewhere now